### DISCUSSION TITLE
Progress report

### DISCUSSION THREAD

**Discussion by leo-benz** [leo-benz/budgetbudget-native#10] [Show and tell]
**Current Progress:**
- Accounts and categories are imported from money money with correct name, icon and hierarchy
- Foundation of main budget layout is implemented
- Basic Settings UI is implemented
<img width="1262" alt="Screenshot 2022-07-27 at 19 51 442" src="https://user-images.githubusercontent.com/3419838/181342301-48a956aa-3718-4a82-a947-17017abb2373.png">
<img width="1012" alt="Screenshot 2022-07-27 at 19 54 35" src="https://user-images.githubusercontent.com/3419838/181341114-0b7d5955-4ba2-41c0-addc-bcf6183bdef6.png">

**Next Steps:**
- Import transactions from MoneyMoney and populate budget view with real spent values
- Create data structure for storing budgets and connect it to the budget view
- Enable importing existing BudgetBudget save files
- Alternating row highlight for main budget view

**Comment by Xiphe** [comment]
WOW 😍

**Comment by diktomat** [comment]
As stated when you announced this, I'm in now that a macOS beta is not required any more. So I've successfully built and run the budget-view branch after removing the iCloud and Push Notification entitlements (don't have a paid developer account).

Before doing any further, it would probably be wise to talk about it, to develop this together and not against each other… What is the current state, last pushed commit being from August? What are the next steps, and where can I step in, given that I'm relatively new to Swift(UI)?

**Reply by Xiphe** [reply]
Awesome 👍 

I was also wondering if i can chime in now that beta is not required anymore. 

Would be interested in joining a call where we can sync up

**Reply by leo-benz** [reply]
Great to see more interest in this.

After the initial work in the summer I was busy with other work and there was no further progress so far. I actually planned to jump back in next week. So far I did not even use any macOS 13 since I didn't risk running the Beta on my Mac. So now some existing parts can make use of the new APIs.

There is more than enough work left to be done both on the UI as well as the functional side. It would make sense to organize the work here on GitHub using issues (and maybe this new projects feature). For more of a realtime collaboration/calls using some Chat Tool would make sense. Any objections for using Discord (supports chats and calls for free)?

**Reply by diktomat** [reply]
It's a Xcode 14 project file though, that's what I meant by requiring beta. Nevertheless, great to hear you got time again! Organizing with issues sounds fine. I personally like Gitter better than Discord, but both probably will work well enough :)

**Reply by Xiphe** [reply]
Cool 👍

I'd go with Gitter and for calls we could use some free tool like whereby.com or google meet.

**Reply by Xiphe** [reply]
Have created a community here: https://gitter.im/BudgetBudget/community
